Makefile

```diff
@@ -92,36 +92,41 @@ init: default
 .PHONY: run
 run: default
 	mkdir -p logs
+	mkdir -p tmp
+	mkdir -p run
 	mkdir -p /tmp/apisix_cores/
-	$(OR_EXEC) -p $$PWD/ -c $$PWD/conf/nginx.conf
+	$(OR_EXEC) -p $$PWD/ -c $$PWD/conf/nginx/nginx.conf
 
 
 ### stop:             Stop the apisix server
 .PHONY: stop
 stop: default
-	$(OR_EXEC) -p $$PWD/ -c $$PWD/conf/nginx.conf -s stop
+	$(OR_EXEC) -p $$PWD/ -c $$PWD/conf/nginx/nginx.conf -s stop
 
 
 ### clean:            Remove generated files
 .PHONY: clean
 clean:
 	rm -rf logs/
+	rm -rf tmp/
+	rm -rf run/
 
 
 ### reload:           Reload the apisix server
 .PHONY: reload
 reload: default
-	$(OR_EXEC) -p $$PWD/  -c $$PWD/conf/nginx.conf -s reload
+	$(OR_EXEC) -p $$PWD/  -c $$PWD/conf/nginx/nginx.conf -s reload
 
 
 ### install:          Install the apisix
 .PHONY: install
 install:
 	$(INSTALL) -d /usr/local/apisix/logs/
-	$(INSTALL) -d /usr/local/apisix/conf/cert
-	$(INSTALL) conf/mime.types /usr/local/apisix/conf/mime.types
+	$(INSTALL) -d /usr/local/apisix/conf/nginx/
```

Comment:
This feature is **not backward compatible** and will affect all users. Is this change really necessary?

Reply:
**Backward compatibility**:
1. The `make install` only create `nginx` directory in `/usr/local/apisix`, have no effect for already exist files such as `/usr/local/apisix/conf/cert/*`, `/usr/local/apisix/conf/mine.types`, `/usr/local/apisix/conf/nginx.conf` etc.
2. We use new `bin/apisix` script to `start, stop, restart` apisix, it will automatically generate `nginx.conf` in `/usr/local/apisix/conf/nginx/` directory.

**Necessary**
The root cause I separate this directory is: We don't suggest to modify the files in `nginx` directory, please only change the `*.yaml`.

** Others **
I thought `make install` is for `production` installation, we currently don't supply any upgrade docs for each releases, if we reach `1.0`(may be a GA release), we may need it for each release later. I'm also working on a `recommended` installation scripts, and this PR is to make configuration more flexible.
Thanks!